model/policy.go

```diff
@@ -125,6 +125,23 @@ func (model Model) GetFilteredPolicy(sec string, ptype string, fieldIndex int, f
 	return res
 }
 
+// CheckPolicy determines whether the specified policy rule can be added to model.
+func (model Model) CheckPolicy(sec string, ptype string, rule []string) bool {
```

Comment:
Rename to `HasPolicyEx`, don't revert the return value

Reply:
If using `HasPolicyEx`, It is best to return `false, err` when the length is invalid, instead of `true`.